APIScrape/scrape.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+require_once "../Utilities/DB.php";
+
+define('API_KEY', 'x-api-key: ec254e44-3996-458b-8522-4933954d8fcd');
+
+$db = new DB();
+$dbConnection = $db->dbConnectToHostOnly();
+
+/**
+ * @param PDO $db Takes DB containing a connection to the database.
+ *  * Creates a new database into which to put the scraped data.
+ * If the database already exists, tears it down and creates a new one.
+ */
+function createDatabase(PDO $db) {
+    try {
+        $sql = "DROP DATABASE IF EXISTS `cats`;
+                CREATE DATABASE `cats`;
+                
+                USE `cats`;
+                CREATE TABLE `breed` (
+                `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+                `breed` varchar(255) NOT NULL DEFAULT '',
+                PRIMARY KEY (`id`),
+                UNIQUE KEY `breed` (`breed`)
+                ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+               
+                CREATE TABLE `img` (
+                `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+                `img_src` varchar(255) NOT NULL DEFAULT '',
+                `breed_id` int(11) unsigned NOT NULL,
+                PRIMARY KEY (`id`),
+                UNIQUE KEY `img_src` (`img_src`)
+                ) ENGINE=InnoDB DEFAULT CHARSET=latin1;";
+        $db->exec($sql);
+        echo "Database successfully initialised.\n";
+    } catch (PDOException $e) {
+        echo $e->getMessage();
+    }
+}
+
+/**
+ * Make API request to get all of the breeds from theCatAPI.
+ * 
+ * @return array of breeds as 4-character breed id => breed name
+ * e.g. "ABYS" => "Abyssinian"
+ */
+function getCatBreeds():array
+{
+    $curl = curl_init();
+    curl_setopt_array($curl, array(
+        CURLOPT_URL => "https://api.thecatapi.com/v1/breeds/",
+        CURLOPT_CUSTOMREQUEST => "GET",
+        CURLOPT_RETURNTRANSFER => true,
+        CURLOPT_HTTPHEADER => [ API_KEY ],
+    ));
+    $response = curl_exec($curl);
+    if($response === false) {
+        $curl_ExecuteError = 'Curl error: ' . curl_error($curl);
```

Comment:
This is the casing you haven't fixed `$curl_ExecuteError` -> `$curlExecuteError`